src/_data/team.json

```diff
@@ -411,11 +411,11 @@
   },
   {
     "name": "Shabnam",
-    "image": "/image/ZadraSol.webp",
+    "image": "https://x.com/iishabbyii/photo",
```

Comment:
also image update in local.

Reply:
I have changed it to githubs profile image